test/integration/tlsroute_test.go

```diff
@@ -150,6 +151,7 @@ func TestTLSRouteEssentials(t *testing.T) {
 			Port:     gatewayv1beta1.PortNumber(ktfkong.DefaultTLSServicePort),
```

Comment:
Why not use the builder since you have already added a `WithTLSConfig` method?

Reply:
👍  This change was made before the introduction of such a builder. I'm gonna use it instead